_inc/client/plans/single-product-backup.jsx

```diff
@@ -155,6 +173,47 @@ function PlanRadioButton( {
 	);
 }
 
+function UpgradeButton( {
+	selectedUpgradeType,
+	upgradesAvailable,
+	billingTimeFrame,
+	currencyCode,
+	onClickHandler,
+} ) {
+	const selectedUpgrade = upgradesAvailable.find( ( { type } ) => type === selectedUpgradeType );
+	if ( ! selectedUpgrade ) {
+		return null;
+	}
+	const { link, name, fullPrice, discountedPrice } = selectedUpgrade;
+	let billingTimeFrameString = '';
+	if ( 'yearly' === billingTimeFrame ) {
+		billingTimeFrameString = 'year';
+	} else if ( 'monthly' === billingTimeFrame ) {
+		billingTimeFrameString = 'month';
+	}
+
+	const { symbol, integer, fraction } = getCurrencyObject(
+		discountedPrice || fullPrice,
+		currencyCode
+	);
+	const price = `${ symbol }${ integer }${ fraction }`;
+
+	return (
+		<div className="single-product-backup__upgrade-button-container">
+			<Button href={ link } onClick={ onClickHandler( selectedUpgradeType ) } primary>
+				{ __( 'Upgrade to %(name) for %(price)/per %(billingTime)', {
+					args: {
+						name,
+						billingTime: billingTimeFrameString,
```

Comment:
This may be complicated for translators as well. Even if we were to make `billingTimeFrameString` translatable when we define it above, translators won't always be able to build a nice phrase for it. Indeed, in some languages "per year" is one word only, and is different from "year".

Reply:
Tweaked in a548f653366f1ebe2aab681c6fdf702f9cc7ca2d